src/wp-includes/cron.php

```diff
@@ -1191,6 +1191,10 @@ function _get_cron_array() {
  * @return bool|WP_Error True if cron array updated. False or WP_Error on failure.
  */
 function _set_cron_array( $cron, $wp_error = false ) {
+	if ( ! is_array( $cron ) ) {
+		$cron = array();
+	}
+
```

Comment:
This change is consistent with other patterns to resolve the PHP 8.1 warning ✅  I agree with this input validation and its implementation ✅